socorro/mozilla_settings.py

```diff
@@ -0,0 +1,309 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
```

Comment:
This module is similar to how Django settings module works. You define `SOCORRO_SETTINGS` in the environment as a Python dotted path to a settings file. Then Socorro loads that into a `Settings` instance and you do stuff with it.

This is the settings for Mozilla's crash ingestion pipeline.